lib/test.py

```diff
@@ -130,6 +131,7 @@ def __init__(self, name, args, suite_ini, params={}, conf_name=None):
         self.is_terminated = False
         self.run_params = params
         self.conf_name = conf_name
+        self.started_time = time.time()
```

Comment:
Hm, it seems we don't need to store this value within the Test instance, right?

Reply:
More I think how to better handle succeeded and failed tests both, more I find the Test.run() method too long already. How about moving time.time() calls outside: into TestSuite.run_test()?

Reply:
Actually we have rerun inside Test.run() method and going outside of it will produce the sum of duration on rerun test instead of the latest duration as it is now.